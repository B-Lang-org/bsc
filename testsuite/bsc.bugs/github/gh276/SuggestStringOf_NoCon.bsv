typedef "Foo" Name;

function Bool isName (String x);
   return (Name == x);
endfunction
