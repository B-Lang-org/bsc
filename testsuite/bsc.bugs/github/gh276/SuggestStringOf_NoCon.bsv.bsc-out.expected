checking package dependencies
compiling SuggestStringOf_NoCon.bsv
Error: "SuggestStringOf_NoCon.bsv", line 4, column 12: (T0003)
  Unbound constructor `Name'. Perhaps `stringOf' is missing?
